src/app/shared/comment-editor/comment-editor.component.ts

```diff
@@ -69,6 +68,56 @@ export class CommentEditorComponent implements OnInit {
     this.commentField.setValidators([Validators.maxLength(this.maxLength)]);
   }
 
+  onKeyPress(event) {
+    if (event.ctrlKey) {
```

Comment:
Normally, mac users will want to use the command `⌘` key instead of ctrl. I think we should check for the user's OS and handle the `⌘+b` or `⌘+i` shortcuts

Reply:
This is a good point, thanks for pointing it out @chunweii . Seems like we can use `metaKey` to handle the `⌘` key, you might want to include this in your code.

Reply:
good point, @chunweii, i've included it in the code, detecting either `metaKey` or `ctrlKey` press, as OS detection seems more complicated than I expected. I think it should be ok as `metaKey` in windows refers to the windows key, and even so, testing on Chrome and Firefox on my windows machine, the browsers did not detect the `metaKey` press. It seems that its been depreciated on Firefox at least (`metaKey` is no longer supported on windows):

![image](https://user-images.githubusercontent.com/67156011/182039748-277121f4-70da-4b99-a1a9-624363fdabe7.png)
From https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey